package.json

```diff
@@ -16,7 +16,7 @@
   },
   "main": "lib/fiware-pep-steelskin",
   "engines": {
-    "node": ">=0.10.8"
+    "node": ">=4.8.4"
```

Comment:
As packages.json has changed... should npm-shrikwrap.json be modified? Not sure this case, as we have not actually changed dependencias (only the node versio)

Reply:
done